backends/dpdk/DpdkXfail.cmake

```diff
@@ -96,6 +96,16 @@ p4c_add_xfail_reason("dpdk"
   testdata/p4_16_samples/pna-example-tcp-connection-tracking-err.p4
   )
 
+p4c_add_xfail_reason("dpdk"
+  "add_entry is not called from a default action"
+  testdata/p4_16_samples/pna-example-addhit.p4
+  )
+
+p4c_add_xfail_reason("dpdk"
+  "add_entry action cannot be default action"
+   testdata/p4_16_samples/pna-example-miss.p4
+   )
+
```

Comment:
Should these be marked xfail?

Reply:
Yes. -addhit calls add_entry from a non-default_action. -miss calls add_entry with first arg as the name of default_action.

Reply:
No, both p4c-dpdk and test infra give same error results. Regarding -error file, most such files have zero size in p4_16_samples_output. 

Also, I see @usha1830 asking Mihai to merge her PRs, so I don't think she has merge permission.

Reply:
If she can't merge it in, I still want to give her a chance to look once more before someone else does, since further changes have been made since she approved.

Reply:
> Right, but I am pretty sure that is why you check in `.p4-error` files with the expected stderr output from p4c-dpdk for those, and the test infra checks that. At least for back ends other than DPDK, xfail means you wish the test case gives no errors, but it currently does for reasons of bugs whose fixes are not checked in, or you wish the test case DOES give an error, but it currently does NOT for reasons of bugs whose fixes are not checked in.
> 
> Does the test infra behave differently than that for DPDK back end tests?

@jafingerhut Test infra does has some issue with comparing the -error files, being tracked using https://github.com/p4lang/p4c/issues/3640. I am working on it. I will cleanup the XFAIL list also post that.